### ISSUE TITLE
Can this be used to find fuzzy matching rows?

### ISSUE THREAD

**Issue by rwarnung** [dgrtwo/fuzzyjoin#23]
I just postd [this question](http://stackoverflow.com/questions/42932160/finding-approximately-matching-rows-of-numerical-data-in-r) which contains the following example:
```
x = matrix( 1:100, nrow = 100, ncol= 4 ,byrow = FALSE)
y = x + matrix( .001 * rnorm(n = 400), nrow = 100, ncol= 4)
z = rbind(x,y)
z = z[sample(nrow(z)),]
```
can fuzzyjoin be used to reduce `z` to `x` (at least as a estimate)?

**Comment by Yeedle** [comment]
I don't think this is in fuzzyjoin's scope. Despite its name, joins made with fuzzyjoin are deterministic and are not estimates.
